### ISSUE TITLE
Accessibility - some actions in the toolbar require descriptions

### ISSUE THREAD

**Issue by SmittyShmee** [ckeditor/ckeditor5#12402]
## 📝 Provide detailed reproduction steps (if any)

1. Use Chrome and [Axe DevTool accessibility testing extension](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-GB).
2. Test page with fairly basic version of CK Editor 5 running

### ✔️ Expected result

No accessibility errors.

### ❌ Actual result

Many of the toolbar buttons flag the "Ensures buttons have discernible text" error. 

### ❓ Possible solution

According to Deque (the author of the plugin), you can do at least one of the following per button:
- Element does not have inner text that is visible to screen readers
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has no title attribute
- Element's default semantics were not overridden with role="none" or role="presentation"

## 📃 Other details

Affected buttons: 
- unordered list styles split button
- ordered list styles split button 
- insert image split button
- highlight button
- highlight split button

Supporting Images:
- Affected Items (circled in orange): https://i.imgur.com/srU6GNO.png
- Result that the Axe DevTools shows for one of the buttons: https://i.imgur.com/uizCb6w.png

Technical Details:
* Browser: Chrome 105.0.5195.54 (64bit)
* Axe DevTools Chrome Plugin: v4.36.2
* OS: Win 10
* First affected CKEditor version: 5
* CK Editor Plugins:
** "@ckeditor/ckeditor5-alignment": "^34.1.0",
** "@ckeditor/ckeditor5-autoformat": "^34.1.0",
** "@ckeditor/ckeditor5-autosave": "^34.1.0",
** "@ckeditor/ckeditor5-basic-styles": "^34.1.0",
** "@ckeditor/ckeditor5-block-quote": "^34.1.0",
** "@ckeditor/ckeditor5-dev-utils": "^30.1.5",
** "@ckeditor/ckeditor5-dev-webpack-plugin": "^30.1.5",
** "@ckeditor/ckeditor5-editor-classic": "^34.1.0",
** "@ckeditor/ckeditor5-essentials": "^34.1.0",
** "@ckeditor/ckeditor5-find-and-replace": "^34.1.0",
** "@ckeditor/ckeditor5-font": "^34.1.0",
** "@ckeditor/ckeditor5-heading": "^34.1.0",
** "@ckeditor/ckeditor5-highlight": "^34.1.0",
** "@ckeditor/ckeditor5-horizontal-line": "^34.1.0",
** "@ckeditor/ckeditor5-image": "^34.1.0",
** "@ckeditor/ckeditor5-indent": "^34.1.0",
** "@ckeditor/ckeditor5-link": "^34.1.0",
** "@ckeditor/ckeditor5-list": "^34.1.0",
** "@ckeditor/ckeditor5-media-embed": "^34.1.0",
** "@ckeditor/ckeditor5-page-break": "^34.1.0",
** "@ckeditor/ckeditor5-paragraph": "^34.1.0",
** "@ckeditor/ckeditor5-paste-from-office": "^34.1.0",
** "@ckeditor/ckeditor5-remove-format": "^34.1.0",
** "@ckeditor/ckeditor5-restricted-editing": "^34.1.0",
** "@ckeditor/ckeditor5-source-editing": "^34.1.0",
** "@ckeditor/ckeditor5-special-characters": "^34.1.0",
** "@ckeditor/ckeditor5-table": "^34.1.0",
** "@ckeditor/ckeditor5-theme-lark": "^34.1.0",
** "@ckeditor/ckeditor5-typing": "^34.1.0",
** "@ckeditor/ckeditor5-word-count": "^34.1.0",
** "@ckeditor/ckeditor5-html-support": "^34.1.0",
** "@ckeditor/ckeditor5-upload": "^34.1.0"

**Comment by Comandeer** [comment]
Did you test it in CKEditor 5 v 35.1.0? I've tested it on an [example in our docs](https://ckeditor.com/docs/ckeditor5/latest/examples/builds-custom/full-featured-editor.html) and the issue is no longer present. We shipped several accessibility fixes in v 35.1.0, which should cover this and many other issues.

**Comment by SmittyShmee** [comment]
Thank you @Comandeer, I'll update to 35.1.0 and test it again, and thank you for you quick response.

**Comment by SmittyShmee** [comment]
Thank you @Comandeer, I have confirmed that upgrading to 35.1.0 removes these a11y issues, and I'm not currently detecting any others. Great job.
